python/seldon_core/batch_processor.py

```diff
@@ -258,6 +271,13 @@ def _start_request_worker(
                 batch_id,
             )
             q_out.put(str_output)
+
+        # Setting time interval before the task is marked as done
+        if batch_interval > 0:
```

Comment:
Later we could have default value of `batch_interval` to be `None` and here check 
```
if batch_interval is not None:
    ...
```

(but not sure if it makes sense to adjust now)